lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -127,10 +128,33 @@ def service(name=nil, version = default_api_version)
             service_path,
             @logger,
             @config,
-            get_credentials,
+            make_channel,
```

Comment:
I think you should update the ServiceLookup so this isn't called "updater_proc" there anymore.